scripts/src/java/org/oppia/android/scripts/coverage/RunCoverage.kt

```diff
@@ -95,7 +97,7 @@ class RunCoverage(
     }
   }
 
-  private fun runCoverageForTarget(testTarget: String): List<String>? {
+  private fun runCoverageForTarget(testTarget: String): CoverageReport? {
```

Comment:
TODO: Refactor to throw an exception (possibly)

Reply:
@Rd4dev  please see comment you created.

Reply:
Oops! implemented that, will resolve this comment now.